src/ShopCertification/Response.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Heureka\ShopCertification;
+
+/**
+ * @author Jakub Chábek <jakub.chabek@heureka.cz>
+ */
+class Response
+{
+
+    /**
+     * @var int
+     */
+    public $code;
+
+    /**
+     * @var string
+     */
+    public $message;
+
+    /**
+     * @var string|null
+     */
+    public $description;
+
+    /**
+     * @param string $json
+     *
+     * @throws JsonException
+     * @throws InvalidResponseException
+     */
+    public function __construct($json)
+    {
+        $response = json_decode($json);
+        if (($response instanceof \stdClass) === false) {
```

Comment:
Viz dokumentace: NULL is returned if the json cannot be decoded or if the encoded data is deeper than the recursion limit.

Napriklad toto vraci NULL:
php -r "var_dump(json_decode('{true}')); 

Tohle pak vytvori objekt:
php -r "var_dump(json_decode('{\"0\":\"true\"}'));"